apps/tests/HelloTest/src/hellotest/RemoveFocusedControl.java

```diff
@@ -30,9 +30,9 @@
 import javafx.scene.Group;
 import javafx.scene.Scene;
 import javafx.scene.control.Button;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.robot.Robot;
 import javafx.stage.Stage;
-import com.sun.glass.events.KeyEvent;
-import com.sun.glass.ui.Robot;
 
 public class RemoveFocusedControl extends Application {
```

Comment:
In addition to the bad import, you need to call the Robot() constructor rather than the internal Application::createRobot method.